README.md

```diff
@@ -1,40 +1,75 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+<img src="https://github.com/sylfort/cats-n-cheese/assets/24916160/ca222744-9e82-497e-ad5c-b6412e03491f" alt="Cats n' Cheese Logo" width="250" heigth="250"/>
 
-## Getting Started
+## 🎮 [Play Now](https://cats-n-cheese.vercel.app)
 
-First, run the development server:
+Cats n' Cheese is a multiplayer game where players act as mice stealing cheese from cats, it's all about strategy and fun! 
```

Comment:
The game is a single player not a multiplayer game.

Reply:
Thanks, fixed